Meraki.Api/Data/Dot1x.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Meraki.Api.Data;
+
+/// <summary>
+/// The current setting for dot1x
+/// </summary>
+[DataContract]
+public class Dot1x
+{
+	/// <summary>
+	/// Dot1x Control Direction
+	/// </summary>
+	[ApiAccess(ApiAccess.ReadUpdate)]
+	[DataMember(Name = "controlDirection")]
+	public string? ControlDirection { get; set; }
```

Comment:
This should be an enum according to https://developer.cisco.com/meraki/api-v1/#!update-network-switch-access-policy